src/lightning/fabric/utilities/optimizer.py

```diff
@@ -31,4 +31,4 @@ def _optimizers_to_device(optimizers: Iterable[Optimizer], device: _DEVICE) -> N
 def _optimizer_to_device(optimizer: Optimizer, device: _DEVICE) -> None:
     """Moves the state of a single optimizer to the device."""
     for p, v in optimizer.state.items():
-        optimizer.state[p] = apply_to_collection(v, Tensor, move_data_to_device, device)
+        optimizer.state[p] = apply_to_collection(v, Tensor, move_data_to_device, device, allow_frozen=True)
```

Comment:
@janEbert Before this can be used, we need to bump the version in the requirements file once the lightning-utilities has made a release. cc @Borda

Reply:
You were faster ;p

Reply:
@Borda Is lightning utilities ready to be used here now?